src/components/PopupWithDeleteCard.js

```diff
@@ -0,0 +1,38 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithDeleteCard extends Popup {
+  constructor(popupSelector, handleFormSubmit) {
+    super(popupSelector);
+    this._handleFormSubmit = handleFormSubmit;
+    // this._submitForm
+    this._popupForm = this._popupElement.querySelector(".modal__form");
+    this._handleSubmit = this._handleSubmit.bind(this);
+    this._submitButton = this._popupForm.querySelector(".modal__button");
+    this._submitButtonText = this._submitButton.textContent;
+    this._deleteCardButton = this._popupElement.querySelector(
+      ".modal__delete-card-button"
+    );
+  }
+  renderLoading(isLoading, loading = "Saving...") {
```

Comment:
Great that you'vve implemented the code for rendering loading via the class methods